### DISCUSSION TITLE
Xiaomi MI Smart Kettle Pro

### DISCUSSION THREAD

**Discussion by sim186** [custom-components/ble_monitor#822] [Q&A]
Is someone able to add the Xiaomi Smart kettle Pro to to ble_monitor? I'm trying to add id in home assistant but I was even not able to get the Mac address and the home assistant integration doesn't list it.
![image](https://user-images.githubusercontent.com/18590835/162579202-ddcc3eee-c380-4ca5-ae1f-3d399dab55a9.png)

**Comment by SmartHomeFanatic** [comment]
Hi!

Did you manage to integrate the kettle into your instance? I've got the MAC address but the kettle isn't added to the addon no matter what I do.

**Comment by Ernst79** [comment]
The smart kettle pro can't be supported, as it requires a connection. BLE monitor only passively listens to BLE advertisements, but the kettle requires you to make a connection and ask for the data. This is not possible with BLE monitor. See also https://github.com/custom-components/ble_monitor/issues/398

**Reply by tismofied** [reply]
Sorry if this is not approved to piggyback on someone else's post but is the decision of incompatibility still true for Govee electric kettle H7170?

**Comment by SmartHomeFanatic** [comment]
Damn. I bought the kettle pro when it was on sale and thought that it was the YM-K1501.
You never stop learning. Thanks for your quick reply!
